sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/NoopTracer.java

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.monitor.opentelemetry.exporter;
+
+import com.azure.core.util.Context;
+import com.azure.core.util.tracing.Tracer;
+
+import java.util.Objects;
+
+class NoopTracer implements Tracer {
+
+    static final AutoCloseable NOOP_CLOSEABLE = () -> {
+    };
+
+    static final Tracer INSTANCE = new NoopTracer();
+
+    NoopTracer() {
+    }
+
+    public Context start(String spanName, Context context) {
```

Comment:
Can you add @Override annotation to all of the interface method implementations?

```suggestion
    @Override
    public Context start(String spanName, Context context) {
```